releasenotes/notes/missing-condition-transpile-e2bfb51aa2d3aa00.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    The previously deprecated ``condition`` attribute for :class:`qiskit.dagcircuit.DAGNode` has been removed.
+    It was deprecated in the 0.15.0 release. The :attr:`qiskit.dagcircuit.DAGNode.op.condition` attribute
+    should be used instead.
```

Comment:
I would tweak this a bit, it's the `condition` kwarg on the constructor for DAGNode that was previously deprecated and removed. The `condition` attribute, ie `DAGNode.condition` still exists and works as expected it's just changed under the covers to be property function now that returns the inner op node's condition.

Reply:
Ok. Got it. Will do.

Reply:
I updated the release note, but wasn't sure if there needs to be a 'use instead' in this case. Let me know if it needs refining.